specification/cpim/resource-manager/Microsoft.AzureActiveDirectory/preview/2020-05-01-preview/cpim.json

```diff
@@ -0,0 +1,366 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-05-01-preview",
+    "title": "cpim",
+    "description": "CPIM Service Client "
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
```

Comment:
2 APIs missing - API to list all guestUsages at subscription scope, API to list all guestUsages at resource group scope

Reply:
added